versioned_docs/version-5/migration.mdx

```diff
@@ -0,0 +1,51 @@
+---
+title: 'Migrating from XState v4 to v5'
+---
+
+### Breaking changes
+
+`Machine` to `createMachine`
+
+`cond` is now `guard`
```

Comment:
I think we could make this more of a guide than a list of changes if we flesh each item out a little bit. I was thinking something like this:

````suggestion
#### `Machine` is now `createMachine`

```diff
-import { Machine } from 'xstate';
+import { createMachine } from 'xstate';

-const machine = Machine({});
+const machine = createMachine({});
```

#### `cond` is now `guard`

```diff
 createMachine({
   on: {
     event: {
-     cond: 'someGuard',
+     guard: 'someGuard',
     },
   },
 });
```
````

Make each item a heading (for generating a table of contents with links) and add a diff to make it clear with code what the change means.

I'm building diffs for all of the changes for https://github.com/statelyai/xstate-tools/issues/321 so we could reuse that here as well.

Reply:
Yeah, while this is in draft, I'm just keeping the items we need to expand as "bullet points" (notes to self).